packages/css-engine/src/core/to-value.ts

```diff
@@ -113,6 +113,11 @@ export const toValue = (
   }
 
   if (value.type === "function") {
+    // Right now, we are using function-value only for filter and backdrop-filter functions
+    if (value.hidden === true) {
```

Comment:
I would be ok with this too
```suggestion
    if (value.hidden) {
```